ui/src/app/settings/containers/User/User.js

```diff
@@ -36,8 +39,8 @@ const User = () => {
           </Button>
           <Button
             appearance="negative"
-            element={Link}
-            to={`/users/${user.id}/delete`}
+            onClick={openPortal}
+            style={{ marginLeft: "1rem" }}
```

Comment:
Could you file an issue on Vanilla if the default doesn't work for this case.

Reply:
Yep, it's this one: https://github.com/canonical-web-and-design/vanilla-framework/issues/2408, we'll just have to remember to update this when that gets fixed.